dockerfiles/Dockerfile.pip_ipython_shell

```diff
@@ -0,0 +1,12 @@
+# This install is NOT minimal!
+#   TODO: Minimize the docker container
+from python:3
+MAINTAINER x10an14
+
+WORKDIR /usr/
+
+RUN pip3 install coconut[ipython]
+
+ENTRYPOINT ["coconut"]
+CMD ["--ipython", "console"]
```

Comment:
Aight =) I learn something every day! I'll re-name(/purpose) it into a "pip_develop_shell", so you've got something to test `pip install coconut[develop]`